solar-land/scss/layout/_header.scss

```diff
@@ -0,0 +1,65 @@
+@use "sass:list";
+@import "../../../shared/scss/components/button";
```

Comment:
I recommend to move it to main file

Reply:
Yes, I did this before, but it caused a lot of errors that disrupted my project. That's why I had to write it here to ensure everything works correctly. thank you for your comment.